tvx/lotus/driver.go

```diff
@@ -32,11 +32,11 @@ func (d *Driver) ExecuteMessage(msg *types.Message, preroot cid.Cid, bs blocksto
 
 	actor, err := st.GetActor(msg.From)
 	if err != nil {
-		return nil, cid.Undef, err
+		fmt.Println("from actor not found: ", msg.From)
```

Comment:
this seems like a bad thing? is there a reason we want to continue when no source actor is in the VM?

Reply:
The invalid test needs to get to the testing stage basically.